include/zephyr/bluetooth/conn.h

```diff
@@ -1849,12 +1856,16 @@ struct bt_conn_cb {
 	 * This callback notifies the application that it might be able to
 	 * allocate a connection object. No guarantee, first come, first serve.
 	 *
-	 * Use this to e.g. re-start connectable advertising or scanning.
+	 * The maximum number of simultaneous connections is configured
+	 * by @kconfig{CONFIG_BT_MAX_CONN}, which usually defaults to a single connection.
+	 *
+	 * This is the event to listen for to start a new connection, connectable advertiser
+	 * or scanner, both when the intention is to start it after the system is completely
```

Comment:
```suggestion
	 * This is the event to listen for to start a new connection or connectable advertiser,
	 * both when the intention is to start it after the system is completely
```

Initiating scanning is not directly related to creating a new connection object, and restarting scanning in this, or the disconnected callback, should always work

Reply:
Thank you for clarifying this, I wasn't entirely convinced myself.

This reference to the scanner comes from the existing API documentation:
> Use this to e.g. re-start connectable advertising or scanning.

which I may have misinterpreted.
I'll fix the API doc.